tests/data/product_stability/ubuntu2404.yml

```diff
@@ -60,6 +60,7 @@ platform_package_overrides:
   shadow: login
   sssd: sssd-common
   sssd-ldap: null
+  systemd-journal-upload: systemd-journal-remote
```

Comment:
I'm not sure if this is needed, see previous comment.

Reply:
Removed